static/app/utils/replays/hydrateRRWebRecordingFrames.tsx

```diff
@@ -2,17 +2,6 @@ import type {RecordingFrame} from 'sentry/utils/replays/types';
 import {EventType} from 'sentry/utils/replays/types';
 import type {ReplayRecord} from 'sentry/views/replays/types';
 
-export function recordingStartFrame(replayRecord: ReplayRecord): RecordingFrame {
-  return {
-    type: EventType.Custom,
-    timestamp: replayRecord.started_at.getTime(),
-    data: {
-      tag: 'replay.start',
```

Comment:
There should be other spots, like types.tsx, that reference `replay.start` and could be cleaned up

Reply:
Updated!